blade-egui/src/lib.rs

```diff
@@ -81,12 +88,62 @@ impl GuiTexture {
                 subresources: &blade_graphics::TextureSubresources::default(),
             },
         );
-        Self { allocation, view }
+
+        let sampler = context.create_sampler(blade_graphics::SamplerDesc {
+            name,
+            address_modes: {
+                let mode = match options.wrap_mode {
+                    egui::TextureWrapMode::ClampToEdge => blade_graphics::AddressMode::ClampToEdge,
+                    egui::TextureWrapMode::Repeat => blade_graphics::AddressMode::Repeat,
+                    egui::TextureWrapMode::MirroredRepeat => {
+                        blade_graphics::AddressMode::MirrorRepeat
+                    }
+                };
+                [mode; 3]
+            },
+            mag_filter: match options.magnification {
+                egui::TextureFilter::Nearest => blade_graphics::FilterMode::Nearest,
```

Comment:
we should have helper function to do this mapping, since we are doing it 3 times here